core/spec/helpers/base_helper_spec.rb

```diff
@@ -75,11 +75,18 @@
     let!(:store) { create(:store) }
     let!(:taxon) { create(:taxon) }
     let!(:product) { create(:product) }
+    let!(:standard_page) { create(:cms_standard_page) }
+    let!(:feature_page) { create(:cms_feature_page) }
+    let!(:home_page) { create(:cms_homepage) }
```

Comment:
@MatthewKennedy this tests shold be part of the `cms_helper_spec.rb` not `base_helper_spec.rb`